frontend/packages/console-shared/src/components/health-checks/health-checks-alert.scss

```diff
@@ -0,0 +1,3 @@
+.ocs-health-checks-alert {
+  margin: 0 20px var(--pf-global--spacer--md);
```

Comment:
can we have pf var even for `20px`

Reply:
PF vars are defined in multiples of 8. So not sure if there is a var for 20px.

Reply:
Yes, we don't have PF variable for 20px